<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1C1C1C;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>24</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/more_vert.png);
padding-left: 3px;
padding-right: 3px;
    border: 2px solid #CCCCCC;
	border-top: 0px;
	border-left: 0px;
	border-right: 0px;
    border-color: rgba(80, 80, 80, 255);
    border-radius: 0px;
background-color: #242424;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="show_child">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:400;&quot;&gt;Show child&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator:checked {
    
image: url(://icons/arrow_drop_down.png);
	height:14px;
	width:14px;
}

QCheckBox::indicator:unchecked {
    image: url(://icons/arrow_drop_right.png);
	height:14px;
	width:14px;
}
QCheckBox {
    font: 700 10pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
    height:14px;
    padding-top: 2px;
    padding-bottom:4px;
    color: #E3E3E3;
background-color: #242424;
    padding-left: 4px;
}
QCheckBox:hover {
    background-color: #414956;
    color: white;
}
QCheckBox{
	padding-left:5px;
	border-left: 0px solid black;
border-right: 0px solid black;
border-top: 0px solid black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="gf">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
border-left: None;
border-top: None;
border-right: None;
    padding: 2px;
    color: #E3E3E3;
    background-color: #242424;
height:16px;
width:16px;
}
QToolButton:hover {
    background-color: #414956;
    color: white;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../resources.qrc">
             <normaloff>:/icons/shapes_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/icons/shapes_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="property_class">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variable name and display name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    font-size: 8pt;
    font-family: &quot;Segoe UI&quot;;
    border-top: 0px;
    border-left: 0px;
    border-right: 0px;
    border-bottom: 2px solid rgba(80, 80, 80, 255);
    border-radius: 0px;
    padding: 2px;
    color: #E3E3E3;
    background-color: #242424;
}

QLineEdit:focus {
    border-color: rgba(80, 80, 80, 255);
}

QLineEdit::selection {
    background-color: #414956;
    color: white;
}</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhLatinOnly|Qt::InputMethodHint::ImhLowercaseOnly</set>
           </property>
           <property name="text">
            <string>Class properties</string>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="element_id_display_2">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variable name and display name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    font-size: 8pt;
    font-family: &quot;Segoe UI&quot;;
    border-top: 0px;
    border-left: 0px;
    border-right: 0px;
    border-bottom: 2px solid rgba(80, 80, 80, 255);
    border-radius: 0px;
    padding: 2px;
    color: #6D6D6D;
    background-color: #242424;
}

QLineEdit:focus {
    border-color: rgba(80, 80, 80, 255);
}

QLineEdit::selection {
    background-color: #414956;
    color: white;
}</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhLatinOnly|Qt::InputMethodHint::ImhLowercaseOnly</set>
           </property>
           <property name="text">
            <string>ID:</string>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="element_id_display">
           <property name="focusPolicy">
            <enum>Qt::FocusPolicy::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variable name and display name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    font-size: 8pt;
    font-family: &quot;Segoe UI&quot;;
    border-top: 0px;
    border-left: 0px;
    border-right: 0px;
    border-bottom: 2px solid rgba(80, 80, 80, 255);
    border-radius: 0px;
    padding: 2px;
    color: #6D6D6D;
    background-color: #242424;
}

QLineEdit:focus {
    border-color: rgba(80, 80, 80, 255);
}

QLineEdit::selection {
    background-color: #414956;
    color: white;
}</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhLatinOnly|Qt::InputMethodHint::ImhLowercaseOnly</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="property_class_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Variable name and display name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    font-size: 8pt;
    font-family: &quot;Segoe UI&quot;;
    border-top: 0px;
    border-left: 0px;
    border-right: 0px;
    border-bottom: 2px solid rgba(80, 80, 80, 255);
    border-radius: 0px;
    padding: 2px;
    color: #E3E3E3;
    background-color: #242424;
}

QLineEdit:focus {
    border-color: rgba(80, 80, 80, 255);
}

QLineEdit::selection {
    background-color: #414956;
    color: white;
}</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::InputMethodHint::ImhLatinOnly|Qt::InputMethodHint::ImhLowercaseOnly</set>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
	border-top:0px;
border-left:0px;
border-right:0px;
    height:14px;
    padding-top: 2px;
    padding-bottom:4px;
	background-color: #242424;
    padding-left: 8px;
	padding-right: 8px;
	color: #E3E3E3;
}

QPushButton:hover {
    background-color: #414956;
    color: white;
}


QPushButton:pressed {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
	border-top:0px;
border-left:0px;
border-right:0px;
    height:14px;
    padding-top: 2px;
    padding-bottom:4px;
	background-color: #242424;
    padding-left: 8px;
	padding-right: 8px;
	color: #E3E3E3;
}</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons/delete_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/delete_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copy_button">
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
	border-top:0px;
border-left:0px;
border-right:0px;
    height:14px;
    padding-top: 2px;
    padding-bottom:4px;
	background-color: #242424;
    padding-left: 8px;
	padding-right: 8px;
	color: #E3E3E3;
}

QPushButton:hover {
    background-color: #414956;
    color: white;
}


QPushButton:pressed {
    font: 580 8pt &quot;Segoe UI&quot;;
    border: 2px solid black;
    border-radius: 0px;
    border-color: rgba(80, 80, 80, 255);
	border-top:0px;
border-left:0px;
border-right:0px;
    height:14px;
    padding-top: 2px;
    padding-bottom:4px;
	background-color: #242424;
    padding-left: 8px;
	padding-right: 8px;
	color: #E3E3E3;
}</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons/content_copy_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/content_copy_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_layout">
     <property name="styleSheet">
      <string notr="true">QFrame#frame_layout {
    font-size: 8pt;
    font-family: &quot;Segoe UI&quot;;
    border: 2px solid rgba(80, 80, 80, 255);
	border-top: 0px;
    border-radius: 0px;
	margin-left: 2px;
margin-right: 2px;
}

QFrame:focus {
    border-color: rgba(80, 80, 80, 255);
}

QFrame::selection {
    background-color: #414956;
    color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="layout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
